src/openforms/config/models.py

```diff
@@ -64,6 +64,12 @@ class GlobalConfiguration(SingletonModel):
         default=runtime_gettext(_("Thank you for submitting this form.")),
         validators=[DjangoTemplateValidator()],
     )
+    submission_report_download_link_title = models.CharField(
+        max_length=128,
+        verbose_name=_("Submission download link title"),
+        help_text=_("The title of the link to download the report of a submission."),
+        default=runtime_gettext(_("Download PDF")),
+    )
```

Comment:
shouldn't this field be marked translatable? Admins/form designers should be able to enter the NL and EN text, would be weird to have an entire form in EN and then suddenly this download link is NL.

You should then also be able to drop the `runtime_gettext` with just the normal gettext as the default language is NL anyway.

Reply:
I thought we use `runtime_gettext` to defer evaluation of the translatable strings. Is that not required in the present case?